package/suse-migration-services-spec-template

```diff
@@ -58,23 +57,25 @@ Requires:         suse-migration-services = %{version}-%{release}
 Requires(preun):  systemd
 Requires(postun): systemd
 BuildArch:        noarch
+%define python_subpackage_only 1
+%python_subpackages
```

Comment:
this doesn't do anything when the main package name does not start with `python-` which is the case here. remove again please.

Reply:
grrr, I thought it was already removed